src/EPR.Payment.Facade/Validations/ProducerRegistrationFeesRequestDtoValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using EPR.Payment.Facade.Common.Constants;
+using EPR.Payment.Facade.Common.Dtos.Request.RegistrationFees;
+using FluentValidation;
+
+namespace EPR.Payment.Facade.Validations
+{
+    public class ProducerRegistrationFeesRequestDtoValidator : AbstractValidator<ProducerRegistrationFeesRequestDto>
+    {
+        public ProducerRegistrationFeesRequestDtoValidator()
+        {
+            RuleFor(x => x.ProducerType)
+                .Must(pt => string.IsNullOrEmpty(pt) || pt.ToUpper() == "L" || pt.ToUpper() == "S")
```

Comment:
Can contains

Reply:
Not sure why we need contains here. Only one character is allowed.

Reply:
I mean something like this 


```
var conditions = new List<string>() { str1, str2, str3 };
RuleFor(x => x.Parameter)
  .Must(x => conditions.Contains(x))
  .WithMessage("Please only use: " + String.Join(", ", conditions));
```

Reply:
@nbevdn I'm happy to be corrected here. Will contains not allow anything like "testSmalltest" instead of the value actually be equal to 'small' or 'large'?

@paulshimmen the document says the values would be "small"/"large" or empty string

Reply:
Agree with both comments. I will use contains and adjust to use "small"/"large".

Reply:
> @nbevdn I'm happy to be corrected here. Will contains not allow anything like "test**L**test" instead of the value actually be equal to 'small' or 'large'?
> 
> @paulshimmen the document says the values would be "small"/"large" or empty string

![image](https://github.com/user-attachments/assets/586a4c4f-a80a-4556-b0f8-04b663d4e521)